### ISSUE TITLE
Unexpected argument(s): checkout

### ISSUE THREAD

**Issue by li-dennis** [CircleCI-Public/aws-ecr-orb#206]
### Orb version
circleci/aws-ecr@8.1.2

### What happened
Attempted to upgrade orb from aws-ecr@6.15.3  to 8.1.2 but ran into the following error on build in circleci:
```
#!/bin/sh -eo pipefail
# Error calling workflow: 'build_test_deploy'
# Error calling job: 'aws-ecr/build-and-push-image'
# Unexpected argument(s): checkout
# 
# -------
# Warning: This configuration was auto-generated to show you the message above.
# Don't rerun this job. Rerunning will have no effect.
false
```

### Expected behavior
checkout should work as expected as a command in our `config.yml` and described in the docs 

<img width="905" alt="Screen Shot 2022-06-01 at 3 52 25 AM" src="https://user-images.githubusercontent.com/1820599/171355127-415af89d-6a7c-4ef0-91eb-27c8fbf5d31a.png">

I've attempted downgrading to v8.0.0 as well to no avail

**Comment by umarpansoota** [comment]
I am facing the same issue while attempting to upgrade orb from `aws-ecr@6.12.2` to `8.1.2` but ran into the following error on build in CircleCI:

`#Error calling command: 'aws-ecr/build-image'
#Unexpected argument(s): checkout, ecr-login`

**Comment by szuckerman17** [comment]
I am facing the same problem `Error calling job: 'aws-ecr/build-and-push-image'Unexpected argument(s): checkout` when trying to upgrade from 6.13.0 to 8.1.2

**Comment by maximebeaudoin** [comment]
+1 something is wrong when we want to put checkout to false.

**Comment by gkeiser** [comment]
:+1: I just discovered this, has anyone figured out a work-around?

**Comment by odie5533** [comment]
> +1: I just discovered this, has anyone figured out a work-around?

I workaround it by using the older orb circleci/aws-ecr@7.3.0

**Comment by ProHJess** [comment]
Seems to be fixed in 8.2.1
